resources/views/tickets/newcreate.blade.php

```diff
@@ -1,12 +1,36 @@
 @extends('layouts.requester')
 @section('content')
-    <div class="description comment">
-        <div class="breadcrumb">
-            <a href="{{ url()->previous() }}">{{ trans_choice('ticket.ticket', 2) }}</a>
-        </div>
+
+
+<style>
+.ticket-header{
+    box-shadow: rgba(0, 0, 0, 0.075) 0px 0.125rem 0.25rem;
+}   
+
+.ticket-header-logo{
+    margin: 20px 0px 3px 0;;
+}
+</style>
```

Comment:
Not a fan of Style blocks in templates can we put this in the CSS or SASS. Not sure what this project uses.